### ISSUE TITLE
pageScroll Option missing scroll offset Feature

### ISSUE THREAD

**Issue by magic-77** [FrDH/mmenu-js#1024]
Hi, thanks for this really great Plugin.
I'm using the newest Version 8 for a Page with One-Page Navigation.
Everything works as expected but i need to adjust the scroll offset as i'm using a sticky Header that overlaps the section it is scrolled to.
But i didn't find a Solution for that. Is this possible?

**Comment by FrDH** [comment]
The plugin uses [element.scrollIntoView](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView), which doesn't support an offset.

**Comment by magic-77** [comment]
Hi, yes you're right as i can see at API Docs.
But what about patching "mmenu.pagescroll.ts" to use it this way (replace scrollIntoView with scrollTo)
```
function scrollTo() {
        if (section) {
            let element         = document.getElementById(section.id),
                headerOffset    = options.offsetToTop,
                elementPosition = element.getBoundingClientRect().top,
                offsetPosition  = elementPosition - headerOffset;

            window.scrollTo({
                top: offsetPosition,
                behavior: "smooth"
            });
        }
        section = null;
    }
```
or not replacing "scrollIntoView" and adding a new Function and checking whether the option "offsetToTop" is set or not

**Comment by FrDH** [comment]
Hmm, did not know about the `behavior: "smooth"` option :)
I'll implement it in the next release.

**Comment by FrDH** [comment]
I added a config option `scrollOffset: 0`:
````js
new Mmenu( '#my-menu', {
   // options
}, {
   // configs
   pageScroll: {
      scrollOffset: 0
   }
})
````

**Comment by FrDH** [comment]
should be fixed in 8.2
